Environmental-Test-Chamber/enviornmentalTestChamberPDD.tex

```diff
@@ -0,0 +1,279 @@
+
+% \section{Secondary Objectives}
+% \label{sec:secondary-obj}
+% Secondary Objectives are lower priority or bonus objectives that are significant but not the main focus of the project. This template does not have secondary objectives.
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+Proper and thorough testing is a critical component of mission success. Many SPEX projects are unable to test their designs in proper environmental conditions, meaning flight tests assume additional risk. An ETC would enable risk reduction on projects like HAB and IREC, as well as provide opportunities to replicate industry-standard testing on space-like designs.\@
+
+Some examples of near-term testing benefits include HAB camera shutdown investigation. HAB-CV camera performance, IREC desert heat validation, and \$50Sat orbital thermal cycling. Sensors and electronics have performance curves related to environmental conditions. An ETC would enable fine characterization of these sensors before flight, increasing confidence in the data collected.\@
+
+An ETC would have reduced mission risk for each of those projects and will be useful for future iterations of those types of projects. An ETC would likely prove useful to many other applications in the near to medium future.
+
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text.
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+  At the end of the project, a working Environmental Test Chamber should be delivered. It should be capable of maintaining a near-vacuum for extended periods of time, as well as cycling between standard and vacuum-like pressures at reasonable rates.
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+  The chamber should also be able to subject the test article to thermal variations between -20 C and 50 C. Cooling is considered an advanced feature due to the difficulty of removing thermal energy from the ETC compared to adding thermal energy. 
+
+  An additional feature would be the ability to direct the applied thermal energy to simulate satellite tumbling or orbital cycles. Another metric to consider is the ability to apply the average solar radiation ( 1,368 W/m2) to the working surface of the test article to simulate on-orbit thermal conditions.
```

Comment:
Measuring humidity is very important here. To go below freezing you want to make sure the chamber is dry (by baking out the chamber at higher temps). Rather than putting full faith in the procedure, the ETC should measure internal humidity.